src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -170,8 +169,8 @@ public void setExtensionNode(DiscoveryExtension extensionNode) {
         this.extensionNode = extensionNode;
     }
 
-    DiscoveryNode getOpensearchNode() {
-        return opensearchNode;
+    public DiscoveryNode getLocalNode() {
```

Comment:
Our exceptions for javadocs on getters require the return value match the field name, so the build is failing on lack of javadoc. Which brings up the question, why rename this?  "LocalNode" implies to me that it's the local (to the extension) node, not the node of OpenSearch/ExtensionsOrchestrator.

Reply:
Ill keep the original name